templates/amazon/parallelcluster_supervisord.conf.erb

```diff
@@ -5,12 +5,12 @@
 <% when 'MasterServer' -%>
```

Comment:
Do we still need this 'amazon' declination? It seems the files are now the same, so we can keep just the default